shardingsphere-infra/shardingsphere-infra-util/src/main/java/org/apache/shardingsphere/infra/util/exception/sql/ShardingSphereSQLException.java

```diff
@@ -40,6 +40,11 @@ public ShardingSphereSQLException(final SQLState sqlState, final int vendorCode,
     }
     
     public ShardingSphereSQLException(final String sqlState, final int vendorCode, final String reason, final Object... messageArguments) {
+        this(null, sqlState, vendorCode, reason, messageArguments);
```

Comment:
There is no reason to set cause to null. May be not need to call super().

Reply:
Calling Super(cause) is to transmit parent exception and display in the stacktrace